debezium-core/src/main/java/io/debezium/relational/history/DatabaseHistory.java

```diff
@@ -74,6 +74,16 @@
                                                         + "from processing and storing into schema history evolution.")
                                                 .withValidation(Field::isListOfRegex);
 
+    public static final Field USE_CATALOG_BEFORE_SCHEMA = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "use.catalog.before.schema")
```

Comment:
I don't think this should be a connector option, as it's not needed to be configured by the user but rather is a fixed setting per connector. Or is this actually needed for DB2 somehow? If not I'd suggest to simply add a parameter `useCatalogBeforeSchema` to the `configure()` method.

Reply:
Yes, exactly. Also the invocation of `configure()` in the MySQL connector (which currently does not inherit from `HistorizedRelationalDatabaseConnectorConfig`,  as it eventually should) needs to be adjusted to stick to its current behavior.

Reply:
Hi @gunnarmorling , I changed DatabaseHistory below to use useCatalogBeforeSchema as a parameter to configure() as you suggested.